packages/electron/e2e/utils/electron-utils.ts

```
@@ -280,17 +280,19 @@ export async function mockNotifications(app: ElectronApplication): Promise<void>
 /**
  * Test file operations with temporary files
  */
+import os from 'os'
+import path from 'path'
```

Biome (2.1.2)

[error] Shouldn't redeclare 'path'. Consider to delete it or rename it.

'path' is defined here:


(lint/suspicious/noRedeclare)